jaus_ros_bridge/src/SetFinDeflection.cpp

```diff
@@ -60,38 +67,46 @@ void SetFinDeflection::ProcessData(char* message) {
   int index = 6;                             // read data starts from index 6
   if (_PresenceVector != nullptr) delete _PresenceVector;
   int16_t pv = buildPresenceVector(message, index);
-  // printf("Presence vector is: %d \n", pv);
-
-  _finId = (int8_t)message[index++];  // 1 byte
-  printf("FinID is: %d \n", _finId);
 
+  _finId = (int8_t)message[index++];            // 1 byte
   _deflectionAngle = (int8_t)message[index++];  // 1 byte
-  printf("_deflectionAngle in degree is: %d \n", _deflectionAngle);
-
-  fin_control::SetAngle msg;
-  msg.ID = _finId;
-  msg.angle_in_radians = JausDataManager::degreesToRadians(_deflectionAngle);
-  // int count = 0;
-  // while(count<5)
-  //{
-  _publisher_setAngle.publish(msg);
-  //    count++;
-
-  //    sleep(0.05);
-  //    printf("sleeping for 50ms. count= %d \n", count);
-  //}
-
-  if (_PresenceVector->IsBitSet((int)0)) {
+
+  //  The OCU sends _finID  from 1 to 4
+  if(_finId < 5)
```

Comment:
@patapufate nit: 
```suggestion
  if (_finId <= 4)
```
is less surprising.

Reply:
Fixed in 5cde148